src/NonPhpFile/NonPhpFileProcessor.php

```diff
@@ -61,6 +70,11 @@ public function getSupportedFileExtensions(): array
         return StaticNonPhpFileSuffixes::SUFFIXES;
     }
 
+    public function isActive(): bool
+    {
+        return count($this->activeRectorsProvider->provideByType(PhpRectorInterface::class)) > 0;
```

Comment:
This is the same like in PhpFileProcessor. Is it correct?

Reply:
I am not sure. But i guess so. The NonPhpFileProcessor is depending on some active Php Rector rules. But this FileProcessor is still a blackbox to me. Haven´t used it by now.

Reply:
Ok. Now we have a dedicated Marker Interface NonPhpRectorInterface. Confusion is gone.